src/i18n/langResources/lang.fr.ts

```diff
@@ -0,0 +1,36 @@
+import { ILangResource } from '../../models/i18n';
+import { constants } from '../../constants';
+
+export const langFr: ILangResource = {
+  newVersion: `Bienvenue dans la nouvelle version de ${constants.extensionName}.`,
+  seeReleaseNotes: 'Information à propos des derniers changement',
+  dontShowThis: 'Ne pas afficher ce message la prochaine fois',
+  seeReadme: 'A propos de l\'extension',
```

Comment:
Can you please add something like `(anglais)` to make clear that this readme file is not in French?